101-linq-samples/src/Program.cs

```diff
@@ -3,155 +3,156 @@
-                "max-projection"                => new AggregateOperators().MaxProjection(),
-                "max-grouped"                   => new AggregateOperators().MaxGrouped(),
-                "max-each-group"                => new AggregateOperators().MaxEachGroup(),
-                "average-syntax"                => new AggregateOperators().AverageSyntax(),
-                "average-projection"            => new AggregateOperators().AverageProjection(),
-                "average-grouped"               => new AggregateOperators().AverageGrouped(),
-                "aggregate-syntax"              => new AggregateOperators().AggregateSyntax(),
-                "aggregate-seeded"              => new AggregateOperators().SeededAggregate(),
-
-                "concat-series"                 => new SequenceOperations().ConcatSeries(),
-                "concat-projections"            => new SequenceOperations().ConcatProjection(),
-                "equal-sequence"                => new SequenceOperations().EqualSequence(),
-                "dot-product"                   => new SequenceOperations().DotProduct(),
-
-                "deferred-execution"            => new QueryExecution().DeferredExecution(),
-                "eager-execution"               => new QueryExecution().EagerExecution(),
-                "reuse-query"                   => new QueryExecution().ReuseQueryDefinition(),
-
-                "cross-join"                    => new JoinOperations().CrossJoinQuery(),
-                "group-join"                    => new JoinOperations().GroupJoinQquery(),
-                "cross-group-join"              => new JoinOperations().CrossGroupJoin(),
-                "left-outer-join"               => new JoinOperations().LeftOuterJoin(),
-
-                null                            => RunAll(),
-                _                               => MissingTag(region),
-            };
+                },
+                "groupby-custom-comparer"       => new Groupings().GroupByCustomComparer(),
+                "nested-groupby-custom"         => new Groupings().NestedGroupByCustom(),
+
+                "distinct-syntax"               => new SetOperations().DistinctSyntax(),
+                "distinct-property-values"      => new SetOperations().DistinctPropertyValues(),
+                "union-syntax"                  => new SetOperations().UnionSyntax(),
+                "union-query-results"           => new SetOperations().UnionOfQueryResults(),
+                "intersect-syntax"              => new SetOperations().IntersectSynxtax(),
+                "intersect-different-queries"   => new SetOperations().IntersectQueryResults(),
+                "difference-of-sets"            => new SetOperations().DifferenceOfSets(),
+                "difference-of-queries"         => new SetOperations().DifferenceOfQueries(),
+                "except-syntax"                 => new SetOperations().DifferenceOfSets(),

```

Comment:
This line is the only change to this file. Hopefully it is useful for reviewers.